start.py

```diff
@@ -1,13 +1,43 @@
+import os
 import sys
+import __main__
 
 from PyQt5.QtWidgets import (QApplication, QMessageBox, QSystemTrayIcon)
 from osfoffline.application.main import OSFApp
 from osfoffline import utils
 
 
+def is_only_instance():
+    # Determine if there are more than the current instance of the application
+    # running at the current time.
```

Comment:
This is specific to mac or linux and will not work on windows. (should be noted in docstring)